include/rppdefs.h

```diff
@@ -361,10 +361,13 @@ typedef enum
  */
 typedef enum
 {
-    NCHW,
-    NHWC,
-    NCDHW,
-    NDHWC
+    NCHW,   // BatchSize-Channels-Height-Width
+    NHWC,   // BatchSize-Height-Width-Channels
+    NCDHW,  // BatchSize-Channels-Depth-Height-Width
+    NDHWC,  // BatchSize-Depth-Height-Width-Channels
+    NHW,    // BatchSize-Height-Width -> Used for Audio Kernels except Spectrogram / MelfilterBank
```

Comment:
NHW is generic to 2D
So need not add these comment i feel
`-> Used for Audio Kernels except Spectrogram / MelfilterBank`